pkg/cmd/cli/backup/get.go

```diff
@@ -49,7 +49,7 @@ func NewGetCommand(f client.Factory, use string) *cobra.Command {
 					backups.Items = append(backups.Items, *backup)
 				}
 			} else {
-				backups, err = arkClient.ArkV1().Backups(api.DefaultNamespace).List(metav1.ListOptions{})
+				backups, err = arkClient.ArkV1().Backups(api.DefaultNamespace).List(metav1.ListOptions{LabelSelector: listOptions.LabelSelector})
```

Comment:
also, this can be simplified to just: `arkClient.ArkV1().Backups(api.DefaultNamespace).List(listOptions)` so we're not double-instantiating a `ListOptions` struct. Same for restore.

Reply:
@skriss Got it. I also notice that schedules are in a similar state- shall I go ahead and update them as well?